update-go-version.sh

```diff
@@ -14,6 +14,7 @@ go mod edit -go "${golang_base_version}.0" tools/go.mod
 
 sed -i "s,^GOVERSION = 1.[0-9]\+,GOVERSION = ${golang_base_version}," Makefile
 sed -i "s,^\(FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-\)1.[0-9]\+,\1${golang_base_version}," images/*/Dockerfile
+sed -i "s,^\(FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-\)1.[0-9]\+,\1${golang_base_version}," images/*/Containerfile
```

Comment:
Should be enough to add ` images/*/Containerfile` to the previous line?

Reply:
@cfergeau I think we need both since this update required both in `Dockerfile` and `Containerfile`
```
$ grep -nR  "registry.ci.openshift.org/openshift/release" images/
images/openshift-ci/Dockerfile:4:FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17 AS builder
images/build-e2e/Containerfile:2:FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17 AS builder
images/build-integration/Containerfile:2:FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17 AS builder
images/build/Dockerfile:1:FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17
```

Reply:
I'm suggesting
```
sed -i "s,^\(FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-\)1.[0-9]\+,\1${golang_base_version}," images/*/Dockerfile images/*/Containerfile
```
instead of having 2 lines.

`images/*/{Containerfile, Dockerfile}` should also work.

Reply:
I see we also duplicate the `sed` regexp for `ubi8/go-toolset` below, we can keep your version and clean that up later. 
Do we still need the `ubi8` updates after the switch to the openshift CI images?

Reply:
@cfergeau no, we don't need ubi8 update now so removed it in different commit.